### DISCUSSION TITLE
Threaded dispatcher updates for {crew}

### DISCUSSION THREAD

**Discussion by shikokuchuo** [r-lib/mirai#151] [Ideas]
Now that threaded dispatcher is merged, this is to track changes needed to enable it for {crew}.

Inviting @wlandau.

**Comment by wlandau** [comment]
To directly interact with the `mirai` dispatcher, `crew` uses `daemons()`, `saisei()`, and `status()`. As long as those functions behave the same way with new threaded dispatcher as the existing process dispatcher, I think I will at least be able to start testing `crew` with the former. IIRC the other functions listed at https://github.com/wlandau/crew/blob/efb1f9b190504ab297f6738adde74d857d5e7066/R/crew_package.R#L25-L26 interact with the dispatcher indirectly or not at all.

**Comment by shikokuchuo** [comment]
Closing this for now as the new dispatcher model will require a different set of things.
